src/mod11.js

```diff
@@ -35,9 +43,9 @@ function getDefaultWeights(n) {
     const weightOf = idx => ((idx + 6) % 6) + 2;
 
     return Array
-            .from({length: n}, (val, idx) => idx) // => [0, 1, .., n-1]
-            .reverse()                            // => [n-1, .., 1, 0]
-            .map(val => weightOf(val));           // => [w(n-1), .., 3, 2]
+        .from({length: n}, (_, idx) => idx) // => [0, 1, .., n-1]
+        .reverse()                            // => [n-1, .., 1, 0]
+        .map(val => weightOf(val));           // => [w(n-1), .., 3, 2]
```

Comment:
Comments should be aligned for readability
```suggestion
        .from({length: n}, (_, idx) => idx) // => [0, 1, .., n-1]
        .reverse()                          // => [n-1, .., 1, 0]
        .map(val => weightOf(val));         // => [w(n-1), .., 3, 2]
```

Reply:
Fixed by c240816